systemtest/src/test/java/io/strimzi/systemtest/AbstractClusterIT.java

```diff
@@ -46,6 +47,11 @@
 public class AbstractClusterIT {
 
     private static final Logger LOGGER = LogManager.getLogger(AbstractClusterIT.class);
+    protected static final String ZK_IMAGE = "STRIMZI_DEFAULT_ZOOKEEPER_IMAGE";
+    protected static final String KAFKA_IMAGE = "STRIMZI_DEFAULT_KAFKA_IMAGE";
+    protected static final String CONNECT_IMAGE = "STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE";
+    protected static final String S2I_IMAGE = "STRIMZI_DEFAULT_KAFKA_CONNECT_S2I_IMAGE";
+    protected static final String TO_IMAGE = "STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE";
```

Comment:
We have a new image in town ... the `STRIMZI_DEFAULT_INIT_KAFKA_IMAGE`. It's related to the init-kafka container.